### PR TITLE
Update UserGuide add features

### PR DESCRIPTION
**Pull request by SimJJ96**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -14,20 +14,49 @@
...
+Adds a food item to the fridge.
+
+Format: `add FOOD_NAME /cat FOOD_CATEGORY /exp EXPIRY_DATE /loc LOCATION_IN_THE_FRIDGE
+
+* The `FOOD_NAME` can be the name of a food but not an empty description. 
+* The `FOOD_CATEGORY` can be the basic food groups otherwise it will be categorised as others.
+* The `EXPIRY_DATE` must be in the format `dd-mm-yyyy`.
+* The `LOCATION_IN_THE_FRIDGE` can be a general compartment in a fridge.
+  
+Additional info:
```
Comment (kwokyto) [inline docs/UserGuide.md:33]: Can we leave this as a 1 liner list, instead of a long vertical list?

**Review by kwokyto — APPROVED** [review]

File: docs/UserGuide.md [hunk 2]
```diff
@@ -14,20 +14,49 @@
...
+* FRIDGE_DOOR
+* OTHERS
 
 Example of usage: 
 
-`todo n/Write the rest of the User Guide d/next week`
+`add chicken /cat meat /exp 30-06-2021 /loc lower_shelf`
 
-`todo n/Refactor the User Guide to remove passive voice d/13/04/2020`
+`add grouper /cat seafood /exp 04-05-2021 /loc freezer`
```
Comment (Leeyp) [inline docs/UserGuide.md]: Would be nice if we could have sample outputs for these usages

**Review by Leeyp — CHANGES_REQUESTED** [review]
Would be nice if we could have sample output after performing 'add function' so user knows what to expect after adding

**Review by joohwan58 — APPROVED** [review]
LGTM

**Review by Vinci-Hu — APPROVED** [review]
LGTM

**Pull request merged**